Error while checking build/O2Physics/code-check for 454ae0c9aff412bf6fb80c9dc12ffb980893b00a at 2026-08-19 12:49:
```
## sw/BUILD/O2Physics-code-check-latest/log
--
========== List of issues found ==========
++ echo 'Found 9 errors and 0 warnings.'
Found 9 errors and 0 warnings.
++ [[ 9 -gt 0 ]]
++ cat /sw/BUILD/3ce5022bf87e6eaadfd78adc13cf7137da967726/O2Physics-code-check/errors.txt
Common/Tools/StandardCCDBLoader.h:36:1: error: nested namespaces can be concatenated [modernize-concat-nested-namespaces,-warnings-as-errors]
Common/Tools/StandardCCDBLoader.h:57:5: error: 'mMeanVtx' should be initialized in a member initializer of the constructor [cppcoreguidelines-prefer-member-initializer,-warnings-as-errors]
Common/Tools/StandardCCDBLoader.h:58:5: error: 'grpmag' should be initialized in a member initializer of the constructor [cppcoreguidelines-prefer-member-initializer,-warnings-as-errors]
Common/Tools/StandardCCDBLoader.h:59:5: error: 'lut' should be initialized in a member initializer of the constructor [cppcoreguidelines-prefer-member-initializer,-warnings-as-errors]
Common/Tools/StandardCCDBLoader.h:93:41: error: use nullptr [modernize-use-nullptr,-warnings-as-errors]
Common/Tools/TrackPropagationModule.h:58:1: error: nested namespaces can be concatenated [modernize-concat-nested-namespaces,-warnings-as-errors]
Common/Tools/TrackPropagationModule.h:88:3: error: use '= default' to define a trivial default constructor [modernize-use-equals-default,-warnings-as-errors]
Common/Tools/TrackPropagationModule.h:161:17: error: redundant string initialization [readability-redundant-string-init,-warnings-as-errors]
Common/Tools/TrackPropagationModule.h:282:7: error: use auto when initializing with a cast to avoid duplicating the type name [modernize-use-auto,-warnings-as-errors]
++ [[ 0 -gt 0 ]]
++ [[ 9 -gt 0 ]]
++ exit 1
--
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/O2Physics/17197/454ae0c9aff412bf6fb80c9dc12ffb980893b00a/build_O2Physics_code-check/pretty.html).

--- a/Common/Tools/StandardCCDBLoader.h
+++ b/Common/Tools/StandardCCDBLoader.h
@@ -90,7 +90,7 @@ class StandardCCDBLoader
     } else {
       LOGF(info, "GRPMagField object returned nullptr, will attempt alternate method");
 
-      o2::parameters::GRPObject* grpo = 0x0;
+      o2::parameters::GRPObject* grpo = nullptr;
       grpo = ccdb->template getForRun<o2::parameters::GRPObject>(cGroup.grpPath.value, currentRunNumber);
       if (!grpo) {
         LOG(fatal) << "Alternate path failed! Got nullptr from CCDB for path " << cGroup.grpPath << " of object GRPObject for run " << currentRunNumber;

